src/crawl.js

```diff
@@ -45,8 +45,14 @@ async function setupClusterAndCrawl({baselineDir, currentDir, urls, ...argValues
     });
 
     // triggering the screenshot scrapping of the two sites
-    await walkUrl(urls[0], visitedBaseline, baselineDir);
-    await walkUrl(urls[1], visitedCurrent, currentDir);
+    if (urls.length === 2 || (urls.length === 1 && argValues.onlyCrawlBaseline)) {
+      await walkUrl(urls[0], visitedBaseline, baselineDir);
+    }
+    if (urls.length === 2) {
+      await walkUrl(urls[1], visitedCurrent, currentDir);
+    } else if (urls.length === 1 && argValues.onlyCrawlCurrent) {
+      await walkUrl(urls[0], visitedCurrent, currentDir);
```

Comment:
This logic could be cleaned up some so that this is grabbing the currentDir url instead of the first one. Right now if you only pass one url and either -u or -w  it uses that url regardless of if that was defined as baseline or current. Also if you pass two urls and -u or -w it will run a full crawl.